python_modules/dagster/dagster/_core/pipes/subprocess.py

```diff
@@ -68,6 +74,18 @@ def run(
         env: Optional[Mapping[str, str]] = None,
         cwd: Optional[str] = None,
     ) -> Iterator[PipesResult]:
+        """Run a subprocess with in a pipes session.
+
+        Args:
+            command (Union[str, Sequence[str]]): The command to run. Will be passed to `subprocess.Popen()`.
+            context (OpExecutionContext): The context from the executing op or asset.
+            extras (Optional[PipesExtras]): An optional dict of extra parameters to pass to the subprocess.
```

Comment:
I was unsure of how much detail to go into for a lot of args, since they appear in a lot of places multiple times.

My thinking was to be terse with arg descriptions and for people to go to our prose docs for a thorough description of how the pieces work. Do you think the param descs should be more detailed?

Reply:
Yeah good point, let's leave an extended explanation to docs.